src/Resources/views/CRUD/base_array_macro.html.twig

```diff
@@ -8,17 +8,42 @@ For the full copyright and license information, please view the LICENSE
 file that was distributed with this source code.
 
 #}
-{% macro render_array(value, inline) %}
-    {% from _self import render_array %}
-    {% for key, val in value %}
-        {% if val is iterable %}
-            [{{ key }} => {{ render_array(val, inline) }}]
-        {% else %}
-            [{{ key }} => {{ val }}]
-        {% endif %}
+{% macro render_array(value, inline, values_only, translate, translation_domain, translation_prefix, translate_key) %}
+    {% spaceless %}
+        {% if not inline %}<ul>{% endif %}
+        {% from _self import render_array %}
+        {% for key, val in value %}
+            {% if inline == true %}
+                [
+            {% else %}
+                {% if val is not iterable or not values_only %}
```

Comment:
I would have added an `<li>` even if it's iterable.

If you don't ['a', 'b', ['c'], ['d']] and  ['a', 'b', ['c', 'd']] will render the same way:
```
- A
- B
    - C
    - D
```

And why the check on values_only ?
I still want the ul/li if I use `values_only`.

Reply:
How it looks as now:

![image](https://user-images.githubusercontent.com/617002/78592992-9329e500-784e-11ea-8165-2430748a5bdc.png)

And only with `{% if val is not iterable %}` rule:

![image](https://user-images.githubusercontent.com/617002/78593057-ad63c300-784e-11ea-87cf-0f30c869449c.png)

The designer inside me decide that the first variant looks better 😆  But we can simplify, of course.

Reply:
![image](https://user-images.githubusercontent.com/617002/78592992-9329e500-784e-11ea-8165-2430748a5bdc.png)

Is ok for me.

But If you use `values_only => true`. My issue is that, when I see this
![Image](https://user-images.githubusercontent.com/617002/78580138-a6cb5080-783a-11ea-8ac9-b34721a26ac3.png)

I don't know if it's
```
['first option', 'second option', ['first option', 'second option']] 
```
Or
```
['first option', 'second option', ['first option'], ['second option']] 
```

If you remove both the `val is not iterable` and the `not values_only` key, you can now see the diff between my two examples.

It should render
![image](https://user-images.githubusercontent.com/9052536/78594249-39262100-7848-11ea-98ed-41f38b8d0c29.png)
And
![image](https://user-images.githubusercontent.com/9052536/78594444-9c17b800-7848-11ea-9243-f857fea50a2a.png)